core/citrus-api/src/test/java/org/citrusframework/spi/ClassPathResourceResolverTest.java

```diff
@@ -0,0 +1,114 @@
+package org.citrusframework.spi;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
```

Comment:
you're mixing JUnit with TestNG. I think this test won't run with Maven, because it almost for sure has no JUnit runner. please double-check this.

Reply:
it is planned to migrate: https://github.com/citrusframework/citrus/issues/876. we've postponed it from 3.0 however, because it is a huge task!